src/components/SearchBox.js

```diff
@@ -57,20 +59,45 @@ class SearchBox extends Component {
 
   componentDidMount() {
     this._initSearchBase();
+    if (this.props.URLParams) {
+      this.setValue({
+        value: this.getSearchTerm(this.props.currUrl),
```

Comment:
What is `currUrl` ? Is this the correct name ?

Reply:
yes, it's the correct name. currUrl as in the URL in the browser tab

Reply:
Shouldn't it be `currentURL` ? I don't think we will need to pass this from the component as a prop this should be tracked on lib side as we do this in Reactivesearch.

Reply:
it's part of the library: https://github.com/appbaseio/react-searchbox/blob/a44ed57bf83fd73d22cc915a7f2efc9596cde623/src/index.js#L17.
The implementation is complex in RS, with the current implementation it's simple and I don't see any side-effects or anti-pattern